.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
+drahtbot/target
```

Comment:
nit: maybe this can be renamed from drahtbot to review_summary, to indicate what it does in the name? Also, to avoid confusion with the collection of scripts with the same name.

Reply:
In the future, I think it would be nice to move the current scripts into this webhook framework which is why I named it drahtbot. I plan later to add the conflicting list of PRs in the summary comment (there's a TODO in the code).

Reply:
Not sure if it is useful to move everything into the webhook framework. For example conflicts will need to be re-calculated on every push to master. However, the complexity is N^2 in the number of pulls for that case, so it is simply not possible to do in a real-time fashion. Given that it is impossible in real-time, and given that the update rate needs to be limited, it might as well be done independently of a webhook.

Reply:
I see. Maybe `webhook_features` is more appropriate then? Labels for example could be moved to this webhook framework so that whenever a PR is created, labels are applied, instead of rescanning every PRs. Same for the "Needs rebase" label.